app/scripts/components/common/cookie-consent/index.tsx

```diff
@@ -0,0 +1,127 @@
+import React, { useState, useEffect } from 'react';
+
+import {
+  Alert,
+  Button,
+  ButtonGroup,
+  Link,
+  Icon
+} from '@trussworks/react-uswds';
+import './index.scss';
+
+interface CookieConsentProps {
+  title: string;
+  copy: string;
+  onFormInteraction: () => void;
+}
+
+export const CookieConsent = ({
+  title,
+  copy,
+  onFormInteraction
+}: CookieConsentProps) => {
+  const [cookieConsentResponded, SetCookieConsentResponded] = useState(Boolean);
+  const [cookieConsentAnswer, SetCookieConsentAnswer] = useState(Boolean);
+  const [closeConsent, setCloseConsent] = useState(Boolean);
```

Comment:
Interesting, never initialized state like this. Is there a benefit to doing it like this? Looked into it and looks like it'd just be more beneficial to be declarative. `const [cookieConsentResponded, SetCookieConsentResponded] = useState<boolean>(false);`